tests/test_events.py

```diff
@@ -1,62 +1,8 @@
 from pytest import mark
 from pytest import raises
 
-from ppb.eventlib import BadEventHandlerException
-from ppb.eventlib import camel_to_snake
-from ppb.eventlib import EventMixin
-
-
-def test_eventmixin():
-    passed_bag = None
-    passed_fire = None
-
-    class Spam:
-        pass
-
-    class Eventable(EventMixin):
-        def on_spam(self, bag, fire_event):
-            nonlocal passed_bag, passed_fire
-            passed_fire = fire_event
-            passed_bag = bag
-
-    bag = Spam()
-    fire_event = lambda: None
-
-    e = Eventable()
-
-    e.__event__(bag, fire_event)
-    assert bag is passed_bag
-    assert fire_event is passed_fire
-
-
-def test_event_mixin_with_bad_signature():
```

Comment:
Not a fan of losing this test, but I also can't think of a good way to handle it.

Reply:
Yup, I think I need to puzzle on how to get it. It's a lot _heavier_ of a test now.